### PR TITLE
[converter] add TensorScatterUpdate op to converter and executor

### PR DESCRIPTION
**Pull request by pyu10055**
To see the logs from the Cloud Build CI, please join either our [discussion](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs) or [announcement](https://groups.google.com/a/tensorflow.org/forum/#!forum/tfjs-announce) mailing list.

- - -
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/7246)

fixed https://github.com/tensorflow/tfjs/issues/6709
This the converter side for the op.

### CODE REVIEW

**Review by Linchenn — APPROVED** [review]
LGTM! 

I think after this PR, TFJS could run models with TensorScatterUpdate op. I am curious, does this PR also enables model converter to convert models with TensorScatterUpdate op?

> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/7246#-:-NKzWGUU2v4gnc4bA2bJ:bmr1dnd)* status: :shipit: complete! 1 of 1 approvals obtained (waiting on @mattsoulanille)

File: tfjs-converter/yarn.lock [hunk 1]
```diff
@@ -115,10 +115,10 @@
   resolved "https://registry.yarnpkg.com/@types/webgl-ext/-/webgl-ext-0.0.30.tgz#0ce498c16a41a23d15289e0b844d945b25f0fb9d"
   integrity sha512-LKVgNmBxN0BbljJrVUwkxwRYqzsAEPcZOe6S2T6ZaBDIrFp0qu4FNlpc5sM1tGbXUYFgdVQIoeLk1Y1UoblyEg==
 
-"@webgpu/types@^0.1.16":
-  version "0.1.21"
-  resolved "https://registry.yarnpkg.com/@webgpu/types/-/types-0.1.21.tgz#b181202daec30d66ccd67264de23814cfd176d3a"
-  integrity sha512-pUrWq3V5PiSGFLeLxoGqReTZmiiXwY3jRkIG5sLLKjyqNxrwm/04b4nw7LSmGWJcKk59XOM/YRTUwOzo4MMlow==
+"@webgpu/types@0.1.16":
```
Comment (qjia7) [inline tfjs-converter/yarn.lock:118]: Why it's `0.1.16` not `0.1.21`? The places we used are all `0.1.21`. See https://github.com/tensorflow/tfjs/blob/master/package.json#L29 and https://github.com/tensorflow/tfjs/blob/master/tfjs-core/package.json#L44

Comment (pyu10055) [reply tfjs-converter/yarn.lock]: updated, thanks

**Comment by pyu10055** [comment]
> 

yes, the op_list json file ensure converter the op is supported.

**Comment by Linchenn** [comment]
> > 
> 
> yes, the op_list json file ensure converter the op is supported.

Thanks!

**Review by mattsoulanille — APPROVED** [review]
LGTM

**Pull request merged**